resources/dicts/thoughts/dead/darkforest/kitten.json

```diff
@@ -9,15 +9,27 @@
             "Feels hopeless, not understanding what {PRONOUN/m_c/subject} did wrong",
             "Closes {PRONOUN/m_c/poss} eyes, hoping to see the stars once they're opened",
             "Is terrified of the shadows surrounding {PRONOUN/m_c/object}",
-            "Feels as if {PRONOUN/m_c/subject} {VERB/m_c/have/has} been abandoned"
+            "Feels as if {PRONOUN/m_c/subject} {VERB/m_c/have/has} been abandoned",
+            "Misses the warmth of the nursery",
+            "Feels like {PRONOUN/m_c/possessive} innocent is gone",
+            "Wishes {PRONOUN/m_c/object} parent could comfort {PRONOUN/m_c/object}",
+            "Wishes chaos to the living",
+            "Wonders how {PRONOUN/m_c/possessive} bright future was taken away",
+            "Wishes {PRONOUN/m_c/subject} could have lived longer to bring destruction to the clan",
+            "Has an evil look despite {PRONOUN/m_c/object} young age",
+            "Plays with the goop on {PRONOUN/m_c/possessive} paws"
         ]
     },
     {
         "id": "dead_kit_to_dead_other_kit",
         "thoughts": [
             "Huddled up close to r_c",
             "Is comforting r_c despite {PRONOUN/m_c/poss} own fear",
-            "Is being comforted by r_c"
+            "Is being comforted by r_c",
+            "Is playing with r_c despite the dark surroundings",
```

Comment:
sounds a little awkward for some reason.. i would maybe word as "in the murky forest" or something. thats just me though